src/NHibernate/Persister/Entity/AbstractEntityPersister.cs

```diff
@@ -1427,7 +1432,7 @@ public virtual object InitializeLazyProperty(string fieldName, object entity, IS
 					lazyIndexes[i] = descriptor.LazyIndex;
 				}
 			}
-
+			
```

Comment:
This line should be reverted.
```suggestion

```

Reply:
By revert I meant to remove the added tabs and leave the new line as it was prior this PR. `InitializeLazyProperties` method must have the same structure as its async counterpart  `InitializeLazyPropertiesAsync`, so that when the async generator is re-run there will be no changes. Now when running the async generator it will make a change in the `InitializeLazyPropertiesAsync`, which we want to avoid.

Reply:
I only committed your suggestion which removed the line. 
Should be fixed now. Thanks!